phpunit.xml.dist

```diff
@@ -50,9 +50,9 @@
         <!-- Postgres
         <env name="db_dsn" value="postgres://localhost/cake_test?timezone=UTC"/>
         -->
-        <!-- Mysql
-        <env name="db_dsn" value="mysql://localhost/cake_test?timezone=UTC"/>
-        -->
+
+        <env name="db_dsn" value="mysql://root:root@localhost/cake_test?timezone=UTC"/>
+
```

Comment:
please don't change the  `phpunit.xml.dist` file 
if you need these changes make a copy of this file as  `phpunit.xml` in your local